packages/react-core/src/next/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+import { css } from '@patternfly/react-styles';
+import { Menu, MenuContent, MenuProps } from '../../../components/Menu';
+import { Popper } from '../../../helpers/Popper/Popper';
+
+export interface DropdownProps extends MenuProps {
+  /** Anything which can be rendered in a dropdown. */
+  children?: React.ReactNode;
+  /** Classes applied to root element of dropdown. */
+  className?: string;
+  /** Renderer for a custom dropdown toggle. Forwards a ref to the toggle. */
+  toggle: (toggleRef: React.RefObject<any>) => React.ReactNode;
+  /** Flag to indicate if menu is opened.*/
+  isOpen?: boolean;
+  /** Function callback called when user selects item. */
+  onSelect?: (event?: React.MouseEvent<Element, MouseEvent>, itemId?: string | number) => void;
+  /** Callback to allow the dropdown component to change the open state of the menu that is set with the isOpen prop .
+   * If this is not provided the component will not close the menu when tab or escape are pressed. */
+  onIsOpenChange?: (isOpen: boolean) => void;
```

Comment:
Per convo on slack, could this be changed to `onOpenChange` so it references less the state and more the behavior?

Description could be something like `Callback to prompt when the menu should be opened or closed. Triggered by clicking outside of the menu, or by hitting either tab or escape.`